GNUmakefile

```diff
@@ -244,3 +250,466 @@ deploy-images:
+BUCKET_NAME:=demo
+.PHONY: create-minio-bucket
+create-minio-bucket:
+	$(eval MINIO_NODEPORT:=$(shell (kubectl get service/minio -o yaml --context=$(DST_CONTEXT) | grep nodePort | cut -c15-)))
+	$(eval MINIO_SERVER_ADDRESS:=$(DST_CLUSTER_IP):$(MINIO_NODEPORT))
+
+	@echo "Creating bucket: $(BUCKET_NAME) in the minio server...."
+	@docker run                                                                                                       \
+			-i                                                                                                  	  \
+			--rm                                                                                                	  \
+			--net=host                                                                                          	  \
+			-u $$(id -u):$$(id -g)                                                                              	  \
+			-v $$(pwd):/go/src/$(MODULE)                                                                        	  \
+			-v $$(pwd)/.go/cache:/.cache                                                                        	  \
+			-v $(HOME)/.kube:/.kube                                                                             	  \
+			-v /tmp:/.mc																							  \
+			-w /go/src/$(MODULE)                                                                                	  \
+			--env HTTP_PROXY=$(HTTP_PROXY)                                                                      	  \
+			--env HTTPS_PROXY=$(HTTPS_PROXY)                                                                    	  \
+			--env GO111MODULE=on                                                                                	  \
+			--env GOFLAGS="-mod=vendor"                                                                         	  \
+			--env KUBECONFIG=$(INTERNAL_CONFIG)                                                                 	  \
+			--env REGISTRY=$(REGISTRY)                                                                          	  \
+			--env MINIO_ACCESS_KEY=$(MINIO_ACCESS_KEY) 																  \
+			--env MINIO_SECRET_KEY=$(MINIO_SECRET_KEY) 																  \
+			$(DEV_IMAGE)                                                                                              \
+			/bin/bash -c '                                                                                            \
+				kubectl wait --for=condition=READY pods --all --timeout=5m --context=$(DST_CONTEXT)                && \
+				sleep 20 																						   && \
+				mc config host add my-minio http://$(MINIO_SERVER_ADDRESS) $(MINIO_ACCESS_KEY) $(MINIO_SECRET_KEY) && \
+				mc mb my-minio/$(BUCKET_NAME)  																		  \
+			'
+	@echo "Done"
+
+# Install necessary Kubemove resources
+# Example: make prepare REGISTRY=<your docker registry>
+.PHONY: prepare
+prepare: register-crds create-rbac-resources deploy-mp-ctrl deploy-me-ctrl deploy-ds-ctrl deploy-minio create-minio-bucket
```

Comment:
Can we limit `prepare` to installation of kubemove only. So We will have
`deploy_kubemove` : install kubemove crd, namespace, rbac, controllers
`purge_kubemove` : remove crd, namespace, rbac, controllers
MinIO related stuff we can remove from makefile and have it under ./examples/minio.

Reply:
Done. Also, Minio resources has been moved in `elasticsearch-plugin` repo.